client/app/scripts/charts/__tests__/nodes-layout-test.js

```diff
@@ -131,6 +131,18 @@ describe('NodesLayout', () => {
       edges: fromJS({
         [edge('n1', 'n4')]: {id: edge('n1', 'n4'), source: 'n1', target: 'n4'}
       })
+    },
+    layoutProps: {
+      nodes: fromJS({
+        n1: {id: 'n1', label: 'lold', labelMinor: 'lmold', rank: 'rold'},
```

Comment:
First run ignores the cache and will always have a fresh `x` & `y` due to the `runLayout()`. That's why I store the result of the first run in `first` and then make sure in the second run they are not overwritten by `layoutProps2`.